src/app/component/ubs-admin/ubs-admin-table/ubs-admin-table.component.html

```diff
@@ -72,6 +72,6 @@
 
 
 
- 
- 
+
+
```

Comment:
please let us use correct formatting, and remove unnecessary  lines

Reply:
I saw that, but it's not my file, so conflicts can appear because of merging if I change something there.